README.md

```diff
@@ -47,15 +47,21 @@ would go something like this:
 <script src="https://unpkg.com/@adobe/reactor-sdk/dist/reactor-sdk.min.js"></script>
 <script>
   const tok = 'Your Access Token';
+  const orgId = 'Your Org Id';
   const url = 'https://reactor.adobe.io';
-  const reactor = new window.Reactor(tok, { reactorUrl: url });
+  const reactor = new window.Reactor(tok, {
+    reactorUrl: url,
+    customHeaders: {orgId: orgId} // This is only required if you are provisioned for more than one org
```

Comment:
change to real header name. consider updating the comment to explain they can pass any headers here?